### PR TITLE
Welcome Page

### PR DESCRIPTION
**Pull request by Benjamin-Frost**
Closes: #4

The page is very basic for now, but it should be sufficient for the registration :)

### CODE REVIEW

File: public/locales/de/translation.json [hunk 1]
```diff
@@ -57,23 +57,20 @@
...
-      "body": "Besucht die Webseite von WorldSkills Germany"
-    },
-    "ytCard": {
-      "title": "YouTube",
-      "body": "Zu unserem YouTube-Kanal"
+    "title": "Willkommen bei den Bundesleistungszentren der Softwareentwicklung",
+    "general": {
+      "title": "Allgemeine Informationen",
+      "subtitle": "Wer sind wir uns was ist unser Ziel?",
+      "text": "Die WorldSkills Germany - Bundesleistungszentren haben die Aufgabe, zusätzlich zu den Trainings nach WorldSkills-Prinzipien für eine möglichst breite Teilnehmerschaft auch die zukünftigen Mitglieder der deutschen Nationalmannschaft und somit Teilnehmer an internationalen Wettbewerben zu trainieren. Sie werden pro Skill-Disziplin exklusiv nur einmal ernannt. Die Ernennung erfolgt durch die Skills-Paten (Verbände, Unternehmen, Institutionen) gemeinsam mit WorldSkills Germany."
```
Comment (JonasWanke) [inline public/locales/de/translation.json:64]:
```suggestion
      "text": "Die Bundesleistungszentren von WorldSkills Germany haben die Aufgabe, zusätzlich zu den Trainings nach WorldSkills-Prinzipien für eine möglichst breite Teilnehmerschaft auch die zukünftigen Mitglieder der deutschen Nationalmannschaft und somit Teilnehmer an internationalen Wettbewerben zu trainieren. Sie werden pro Skill-Disziplin exklusiv nur einmal ernannt. Die Ernennung erfolgt durch die Skills-Paten (Verbände, Unternehmen, Institutionen) gemeinsam mit WorldSkills Germany."
```

File: public/locales/de/translation.json [hunk 2]
```diff
@@ -57,23 +57,20 @@
...
-      "title": "WorldSkills Germany",
-      "body": "Besucht die Webseite von WorldSkills Germany"
-    },
-    "ytCard": {
-      "title": "YouTube",
-      "body": "Zu unserem YouTube-Kanal"
+    "title": "Willkommen bei den Bundesleistungszentren der Softwareentwicklung",
+    "general": {
+      "title": "Allgemeine Informationen",
+      "subtitle": "Wer sind wir uns was ist unser Ziel?",
```
Comment (JonasWanke) [inline public/locales/de/translation.json:63]:
```suggestion
      "subtitle": "Wer sind wir und was ist unser Ziel?",
```

File: public/locales/en/translation.json [hunk 3]
```diff
@@ -57,23 +57,20 @@
...
-      "body": "Visit the website of WorldSkills Germany"
-    },
-    "ytCard": {
-      "title": "YouTube",
-      "body": "To our YouTube channel"
+    "title": "Welcome to the national centers of software development",
+    "general": {
+      "title": "General Information",
+      "subtitle": "Who are we and what is our goal?",
+      "text": "The WorldSkills Germany - National Training Centres have the task of training future members of the German national team and thus participants in international competitions, in addition to training according to WorldSkills principles for as broad a group of participants as possible. They are appointed exclusively only once per skills discipline. The appointment is made by the skills mentors (associations, companies, institutions) together with WorldSkills Germany."
```
Comment (JonasWanke) [inline public/locales/en/translation.json:64]:
```suggestion
      "text": "The National Training Centres by WorldSkills Germany have the task of training future members of the German national team and thus participants in international competitions, in addition to training according to WorldSkills principles for as broad a group of participants as possible. They are appointed exclusively only once per skills discipline. The appointment is made by the skills mentors (associations, companies, institutions) together with WorldSkills Germany."
```

File: public/locales/en/translation.json [hunk 4]
```diff
@@ -57,23 +57,20 @@
...
+    "skills": {
+      "title": "Our Skills",
+      "text": "The Federal Centres of Excellence in Software Development are located at the <0>Albert-Einstein-Gymnasium Neubrandenburg</0>. At this location, <1>Skill 08: Mobile Applications Development</1> and <2>Skill 09: IT Software Solutions for Business</2> are represented. More information on the individual skills can be found on the respective subpages."
     },
-    "fbCard": {
-      "title": "Facebook",
-      "body": "To our Facebook profile"
+    "cta": {
+      "title": "Become part of our national team",
+      "subtitle": "Have you always wanted to take part in a software development competition and compete with the best of the best? Then apply now for the online qualification and get a chance to be part of our national team.",
```
Comment (JonasWanke) [inline public/locales/en/translation.json:72]:
```suggestion
      "subtitle": "Have you always wanted to take part in a software development competition and compete with the best of the best? Then register now for the online qualification and get a chance to be part of our national team.",
```

File: public/locales/de/translation.json [hunk 5]
```diff
@@ -57,23 +57,20 @@
...
+    "skills": {
+      "title": "Unsere Skills",
+      "text": "Die Bundesleistungszentren der Softwareentwicklung haben ihren Standort am <0>Albert-Einstein-Gymnasium Neubrandenburg</0>. An diesem Standort sind <1>Skill 08: Mobile Applications Development</1> und <2>Skill 09: IT Software Solutions for Business</2> vertreten. Mehr Informationen zu den einzelnen Skills sind auf den jeweiligen Unterseiten."
     },
-    "fbCard": {
-      "title": "Facebook",
-      "body": "Zu unserem Facebook-Profil"
+    "cta": {
+      "title": "Werde Teil unseres Nationalteams",
+      "subtitle": "Wolltest du schon immer an einem Wettbewerb im Bereich der Softwareentwicklung teilnehmen und dich mit den Besten der Besten messen? Dann bewirb dich jetzt für die Online-Qualifikation und erhalte eine Chance, Teil unseres Nationalteams zu werden.",
```
Comment (JonasWanke) [inline public/locales/de/translation.json:72]:
```suggestion
      "subtitle": "Wolltest du schon immer an einem Wettbewerb im Bereich der Softwareentwicklung teilnehmen und dich mit den Besten der Besten messen? Dann registriere dich jetzt für die Online-Qualifikation und erhalte eine Chance, Teil unseres Nationalteams zu werden.",
```

**Review by JonasWanke — CHANGES_REQUESTED** [review]
The page looks good, but I only checked the localizations and no other source files

**Review by JonasWanke — APPROVED** [review]

File: src/components/skill/SkillHeading.astro [hunk 6]
```diff
@@ -20,7 +20,7 @@ const { skillNumber, skillName, img } = Astro.props;
     </div>
   </div>
   <div
-    class="relative mx-auto max-w-7xl py-8 px-4 sm:py-16 md:y-32 sm:px-6 lg:px-8"
+    class="relative mx-auto max-w-7xl py-8 px-4 sm:py-16 md:py-32 sm:px-6 lg:px-8"
```
Comment (CodesDoWork) [inline src/components/skill/SkillHeading.astro:23]: This component is very similar to the Hero one. Can you check if we can avoid duplicates? For example, the waves could be an extra file.

File: public/locales/de/translation.json [hunk 7]
```diff
@@ -57,23 +57,20 @@
...
-      "body": "Besucht die Webseite von WorldSkills Germany"
-    },
-    "ytCard": {
-      "title": "YouTube",
-      "body": "Zu unserem YouTube-Kanal"
+    "title": "Willkommen bei den Bundesleistungszentren der Softwareentwicklung",
+    "general": {
+      "title": "Allgemeine Informationen",
+      "subtitle": "Wer sind wir und was ist unser Ziel?",
+      "text": "Die Bundesleistungszentren von WorldSkills Germany haben die Aufgabe, zusätzlich zu den Trainings nach WorldSkills-Prinzipien für eine möglichst breite Teilnehmerschaft auch die zukünftigen Mitglieder der deutschen Nationalmannschaft und somit Teilnehmer an internationalen Wettbewerben zu trainieren. Sie werden pro Skill-Disziplin exklusiv nur einmal ernannt. Die Ernennung erfolgt durch die Skills-Paten (Verbände, Unternehmen, Institutionen) gemeinsam mit WorldSkills Germany."
```
Comment (CodesDoWork) [inline public/locales/de/translation.json:64]: Maybe link WSG

File: public/locales/en/translation.json [hunk 8]
```diff
@@ -57,23 +57,20 @@
...
-      "body": "Visit the website of WorldSkills Germany"
-    },
-    "ytCard": {
-      "title": "YouTube",
-      "body": "To our YouTube channel"
+    "title": "Welcome to the national centers of software development",
+    "general": {
+      "title": "General Information",
+      "subtitle": "Who are we and what is our goal?",
+      "text": "The National Training Centres by WorldSkills Germany have the task of training future members of the German national team and thus participants in international competitions, in addition to training according to WorldSkills principles for as broad a group of participants as possible. They are appointed exclusively only once per skills discipline. The appointment is made by the skills mentors (associations, companies, institutions) together with WorldSkills Germany."
```
Comment (CodesDoWork) [inline public/locales/en/translation.json:64]: Maybe link WSG

**Review by CodesDoWork — APPROVED** [review]

**Pull request merged**